TestOutput/EventHeaderInterceptorLE64.dat.linux.json

```diff
@@ -33,8 +33,8 @@
   { "n": "TestProviderC:huptr", "uptr": "0x10E1", "(18446744073709551615UL)": "0xFFFFFFFFFFFFFFFF", "meta": { "level": 5 } },
   { "n": "TestProviderC:hiL", "iL": "0x77359400", "(-0x7fffffffffffffffL - 1L)": "0x8000000000000000", "meta": { "level": 5 } },
   { "n": "TestProviderC:huL", "uL": "0xEE6B2800", "(0x7fffffffffffffffL * 2UL + 1UL)": "0xFFFFFFFFFFFFFFFF", "meta": { "level": 5 } },
-  { "n": "TestProviderC:f32", "f32": 3.1400001, "1/3": 0.333333343, "NaN": "-nan", "-Inf": "-inf", "meta": { "level": 5 } },
-  { "n": "TestProviderC:f64", "f64": 6.2800000000000002, "1/3": 0.33333333333333331, "NaN": "-nan", "-Inf": "-inf", "meta": { "level": 5 } },
+  { "n": "TestProviderC:f32", "f32": 3.14, "1/3": 0.33333334, "NaN": "-nan", "-Inf": "-inf", "meta": { "level": 5 } },
+  { "n": "TestProviderC:f64", "f64": 6.28, "1/3": 0.3333333333333333, "NaN": "-nan", "-Inf": "-inf", "meta": { "level": 5 } },
```

Comment:
Note: Switched from `snprintf("%.9g", value)` to `std::to_chars(value)`. to_chars is intended to output "shortest string that will parse back to the same value", which is what we want for JSON and most other output scenarios.